src/components/rasa-chatbot.tsx

```diff
@@ -0,0 +1,262 @@
+import React, { useLayoutEffect } from 'react'
+import { clamp } from 'lodash'
+
+import {
+  Widget,
+  toggle,
+  open,
+  close,
+  show,
+  hide,
+  toggleInputDisabled
```

Comment:
it's kinda odd that this component exports global handlers for those kind of things, kinda breaks the functional principles of React. Does the component not have a more 'react' way of being controlled? (using states and props)

Reply:
Yeah I know it's weird, unfortunately that's all that is exposed :/